cmd/karma/exporter.go

```diff
@@ -23,8 +23,9 @@ var (
 
 	reqDuration = prometheus.NewHistogramVec(
 		prometheus.HistogramOpts{
-			Name: "http_request_duration_seconds",
-			Help: "HTTP request latencies in seconds.",
+			Name:    "http_request_duration_seconds",
+			Help:    "HTTP request latencies in seconds.",
+			Buckets: []float64{0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5}, // prometheus.DefBuckets, but cut off at 0.5 instead of 10 sec
```

Comment:
I doubt that `0.005` is really useful, on the other hand I would assume that `1s` can easily be hit with cpu rate limited instance, how about we drop `0.005` and add `1.0`?

Reply:
Good point!